src/Analysis/Engine/Impl/OverloadResult.cs

```diff
@@ -293,25 +294,16 @@ internal BuiltinFunctionOverloadResult(PythonAnalyzer state, string name, IPytho
             );
         }
 
-        internal override OverloadResult WithoutLeadingParameters(int skipCount = 1) {
-            return new BuiltinFunctionOverloadResult(_projectState, Name, _overload, skipCount, _fallbackDoc);
-        }
+        internal override OverloadResult WithoutLeadingParameters(int skipCount = 1)
+            => new BuiltinFunctionOverloadResult(_projectState, Name, _overload, skipCount, _fallbackDoc);
 
         public override string Documentation {
             get {
-                if (_docTask != null) {
-                    _docTask.Wait(200);
-                }
+                _docTask?.Wait(200);
                 return _doc;
             }
         }
 
-        private void Calculate() {
-            // initially fill in w/ a string saying we don't yet have the documentation
-            _doc = Resources.CalculatingDocumentation;
-            _docTask = Task.Factory.StartNew(DocCalculator);
-        }
-
         public override string ToString() {
             _docTask?.Wait();
```

Comment:
Should this also specify a timeout?

Reply:
Good point. However, this code doesn't seem to make sense... Why wait and then call base anyway?

Reply:
The base class reads Documentation, which is overridden in this class, so potentially when ToString is called, the documentation isn't yet populated, but since the override also has a wait, then removing it is probably okay.